src/app/common/base.component.ts

```diff
@@ -5,9 +5,11 @@ import ProjectLinks from "../project-links";
 import { requireValue } from "./app.helpers";
 import { UnsubscribeOnDestroyAdapter } from "./unsubscribe.ondestroy.adapter";
 import { Observable, map } from "rxjs";
+import { LocalStorageService } from "../services/local-storage.service";
 
 export abstract class BaseComponent extends UnsubscribeOnDestroyAdapter {
     protected activatedRoute = inject(ActivatedRoute);
+    protected localStorageService = inject(LocalStorageService);
```

Comment:
`LocalStorageService` is a very low-level utility to be used that widely. Consider binding to `LoggedUserService` instead to ask the same question.